### ISSUE TITLE
Refactor README.md file

### ISSUE THREAD

**Issue by adbharadwaj** [Murali-group/GraphSpace#372]
README.md is also very long right now. We want to move OS-specific installation instructions to a separate page and rather provide hyperlinks to installation instructions on README page. 

I suggest we should add a table of content which will hyperlink to wiki documentation.

**Comment by adbharadwaj** [comment]
I also feel that we should remove the part about three dummy users.  It creates a lot of confusion among first-time contributors.

```
GraphSpace has three dummy users:

    Username: user1@example.com Password: user1
    Username: user2@example.com Password: user2
    Username: user3@example.com Password: user3

```

**Comment by tmmurali** [comment]
I agree both to refactoring the installation docs and the dummy users. We
should also remove the users from GraphSpace.

**Comment by sanket0211** [comment]
@adbharadwaj Sir and Prof. @tmmurali, 
I agree with you about refactoring README.md. I am taking up this issue and shall submit a patch with the new README.md with the hyperlinks to wiki documentation.
Currently, I suppose the wiki documentation is lacking in information regarding installation. We can shift the installation instructions written in README.md to the wiki page and give hyperlinks to it. 
Secondly, we can also have a link to a video/GIF which shows steps for installation. That shall give more clarity to new users. 
This is the link to the wiki page which lacks information: https://github.com/Murali-group/GraphSpace/wiki/Installation-Instructions
Thank you.

**Comment by sanket0211** [comment]
@adbharadwaj Sir ad Prof. @tmmurali, 
I have submitted a patch - #373 with the new README and also created the wiki pages:
1. https://github.com/Murali-group/GraphSpace/wiki/Running-GraphSpace-Locally
2. https://github.com/Murali-group/GraphSpace/wiki/Running-GraphSpace-on-Apache
Please have a look and let me know if any changes are required. 

Thank you.
